src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/RespondentService.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;

```

Comment:
Should this be under the service package instead of helper?

Reply:
It is helper. So renamed and changed to use @component stereotype